plugins/modules/ovirt_disk.py

```diff
@@ -374,35 +393,13 @@ def transfer(connection, module, direction, transfer_func):
         elif auth.get('ca_file'):
             context.load_verify_locations(cafile=auth.get('ca_file'))
 
-        try:
-            destination_url = urlparse(transfer.transfer_url)
-            transfer_connection = HTTPSConnection(
-                destination_url.hostname,
-                destination_url.port,
-                context=context,
-            )
-            transfer_func(
-                transfer_service,
-                transfer_connection,
-                destination_url,
-                transfer.signed_ticket
-            )
-        except Exception as e:
-            module.warn("Image transfer fail with message: {}.".format(e))
-            module.warn("Retrying trasfer with proxy.")
-            destination_url = urlparse(transfer.proxy_url)
-            transfer_connection = HTTPSConnection(
-                destination_url.hostname,
-                destination_url.port,
-                context=context,
-            )
-
-            transfer_func(
-                transfer_service,
-                transfer_connection,
-                destination_url,
-                transfer.signed_ticket
-            )
+        transfer_connection, url = create_transfer_connection(module, transfer, context)
+        transfer_func(
+            transfer_service,
+            transfer_connection,
+            url,
+            transfer.signed_ticket
```

Comment:
transfer.signed_ticket is not needed since 4.2 when uploading to proxy,
and we never needed when uploading to host.
See this patch explaning the issue and removing it from the SDK.
https://gerrit.ovirt.org/c/108534/

Do not add new code using transfer.signed_ticket, your code will break in 4.4.

Reply:
okay will remove it